configure_log_agent.sh

```diff
@@ -34,7 +34,7 @@ install_system_service() {
     sudo systemctl start monasca-log-agent
 }
 
-generate_config_file() {
+generate_specific_config_file() {
```

Comment:
line 56 curently checks if $FILES is empty, but that code will now never be reached.

Reply:
Done.